script.js

```diff
@@ -19,57 +19,81 @@ function listOnLoad() {
 let outPut = document.getElementById("outPut");
 let spinner = document.getElementById("spinner");
 let input = document.getElementById('inputNum');
-let largerFifty = document.getElementById('largerFifty')
+let largerFifty = document.getElementById('largerFifty');
+let checkBox = document.getElementById('checkBox');
+
+function Fibonacci(x) {
```

Comment:
functions names should not start with Capital letter. You will learn more about it in the future in the course.